README.adoc

```diff
@@ -122,13 +122,28 @@ IMPORTANT: The `id` used in your `settings.xml` *must* be `github-sirius-emfjson
 1. Clone this repository
 2. Build the frontend packages:
 +
+[source,sh]
 ----
 npm ci
 npx turbo run build
 ----
 +
 
 NOTE: In order to run tests, use `npx turbo run coverage`
++
+
+[WARNING]
+====
+To build the package `@eclipse-sirius/sirius-components-diagrams` the `mkdir` command is required.
+
+For windows users, according to the link:https://docs.npmjs.com/cli/v8/using-npm/config#script-shell[npm documentation] the default value for script-shell configuration on windows is `cmd.exe` which does not support the `mkdir` command.
```

Comment:
Windows (capital W, 2 occurrences).

Reply:
or we could replace `mkdir dist/representation` with:
```
node -e 'fs.mkdir("-p dist/representation", { recursive: true } (err) => { if (err) { console.log(err); }})'
```
:smile:

Reply:
Done